src/components/scenes/WcConnectScene.tsx

```diff
@@ -77,6 +77,7 @@ export const WcConnectScene = (props: Props) => {
       })
     } catch (e: any) {
       showError('Failed to connect, try again.')
+      console.error('wcInit error:', e.message)
```

Comment:
Never access `.message` unless you have first cleaned the error. In this case, you can just do `console.error('wcInit error:', e)` and not worry about the cleaning, since that will log the message just the same.